src/cursor/cur_file.c

```diff
@@ -526,7 +526,8 @@ __curfile_close(WT_CURSOR *cursor)
     }
 
 done:
-    API_END_RET(session, ret);
+    __wt_op_timer_stop(session);
```

Comment:
It's ugly that I have to manually call to stop the timer for all the cursor close implementations. However, I don't have a better good idea.